src/main/java/ch/njol/skript/effects/EffReplace.java

```diff
@@ -44,37 +44,42 @@
 @Name("Replace")
 @Description("Replaces all occurrences of a given text with another text. Please note that you can only change variables and a few expressions, e.g. a <a href='./expressions.html#ExprMessage'>message</a> or a line of a sign.")
 @Examples({"replace \"<item>\" in {textvar} with \"%item%\"",
-		"replace every \"&\" with \"§\" in line 1",
-		"# The following acts as a simple chat censor, but it will e.g. censor mass, hassle, assassin, etc. as well:",
-		"on chat:",
-		"	replace all \"kys\", \"idiot\" and \"noob\" with \"****\" in the message",
-		" ",
-		"replace all stone and dirt in player's inventory and player's top inventory with diamond"})
-@Since("2.0, 2.2-dev24 (replace in multiple strings and replace items in inventory), 2.5 (replace first, case sensitivity)")
+	"replace every \"&\" with \"§\" in line 1",
+	"# The following acts as a simple chat censor, but it will e.g. censor mass, hassle, assassin, etc. as well:",
+	"on chat:",
+	"	replace all \"kys\", \"idiot\" and \"noob\" with \"****\" in the message",
+	" ",
+	"replace all stone and dirt in player's inventory and player's top inventory with diamond"})
+@Since("2.0, 2.2-dev24 (replace in multiple strings and replace items in inventory), 2.5 (replace first, case sensitivity), INSERT_VERSION")
 public class EffReplace extends Effect {
 
 	static {
 		Skript.registerEffect(EffReplace.class,
-				"replace (all|every|) %strings% in %strings% with %string% [(1¦with case sensitivity)]",
-				"replace (all|every|) %strings% with %string% in %strings% [(1¦with case sensitivity)]",
-				"replace first %strings% in %strings% with %string% [(1¦with case sensitivity)]",
-				"replace first %strings% with %string% in %string% [(1¦with case sensitivity)]",
-				"replace (all|every|) %itemtypes% in %inventories% with %itemtype%",
-				"replace (all|every|) %itemtypes% with %itemtype% in %inventories%");
+			"replace (all|every|) %strings% in %strings% with %string% [(1¦with case sensitivity)]",
+			"replace (all|every|) %strings% with %string% in %strings% [(1¦with case sensitivity)]",
+			"replace (:first|:last) %strings% in %strings% with %string% [(1¦with case sensitivity)]",
+			"replace (:first|:last) %strings% with %string% in %string% [(1¦with case sensitivity)]",
+			"replace (all|every|) %itemtypes% in %inventories% with %itemtype%",
+			"replace (all|every|) %itemtypes% with %itemtype% in %inventories%");
```

Comment:
```suggestion
			"replace [all|every] %strings% in %strings% with %string% [1:with case sensitivity]",
			"replace [all|every] %strings% with %string% in %strings% [1:with case sensitivity]",
			"replace (:first|:last) %strings% in %strings% with %string% [1:with case sensitivity]",
			"replace (:first|:last) %strings% with %string% in %string% [1:with case sensitivity]",
			"replace [all|every] %itemtypes% in %inventories% with %itemtype%",
			"replace [all|every] %itemtypes% with %itemtype% in %inventories%");
```
while you're here, let's clean up the patterns too.